src/plugins/jartic/archive_downloader.py

```
@@ -0,0 +1,243 @@
+from typing import Any, Dict, List, Optional
+
+import aiohttp
+from bs4 import BeautifulSoup
+from tqdm import tqdm
+
+
+logger = logging.getLogger(__name__)
+
+
+class JARTICArchiveDownloader:
+    def __init__(
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)